src/NuGet.Clients/NuGetConsole.Host.PowerShell/PowerShellHost.cs

```diff
@@ -356,7 +328,53 @@ public void Initialize(IConsole console)
                         ExceptionHelper.WriteErrorToActivityLog(ex);
                     }
                 }
-            });
+
+                return Task.CompletedTask;
+            }
+            );
+        }
+
+        private async Task GetRunspaceAsync(IConsole console)
+        {
+            var result = _runspaceManager.GetRunspace(console, _name);
```

Comment:
This is no longer inside a try-catch block, meaning if it throws (powershell can't be initialised for some reason), it might crash VS, according to the `Initialize` method's catch block's comments.

Reply:
Ok. Added try catch logic same as `Initialize` method.